docs/hps_align.survey.rst

```diff
@@ -0,0 +1,152 @@
+Survey Data Conversion
+======================
+
+.. automodule:: hps_align.survey
+   :exclude-members: SensorEdge, Wire
+   :members:
+   :undoc-members:
+   :show-inheritance:
+   :private-members:
+
+Coordinate Systems
+------------------
```

Comment:
What do you think of putting the image references here (or even in the class doc themselves)?

Generally, I like the idea of putting the documentation and images close to the code, but it might be nice to have the diagrams isolated since they may be referenced from other locations (This is why I put the UVW coordinate diagram on its own page since I felt like it is pretty general.)